shared/src/profileData.ts

```diff
@@ -150,6 +151,7 @@ export const emptyProfileData: ProfileData = {
   nonEssentialRadioAnswers: {},
   elevatorOwningBusiness: undefined,
   communityAffairsAddress: undefined,
+  raffleBingoGames: undefined,
```

Comment:
**Context:** The `raffle-bingo` non-essential question is industry-agnostic, so I added the `raffleBingoGames` property to `ProfileData` instead of `IndustrySpecificData` (or its related structures), which we typically use for industry-specific questions.

Reply:
ok